Gemfile

```
@@ -1,8 +1,11 @@
 source 'http://rubygems.org'
-gem "gosu"
- 
+
+gem 'gosu'
+
 group :test do
-  gem "rspec", ">= 2.1.0"
-  gem "watchr"
-  gem "rcov"
+  gem 'rspec', '>= 2.1.0'
+end
+
+group :dev do
+  gem "rubocop", :require => false
```

```suggestion
  gem 'rubocop', require: false
```
...if you want to go all-in on single quotes and key/value syntax? Either way, this file should be consistent.